README.md

```diff
@@ -141,6 +142,22 @@ add it to your $PATH.
 
 ## Usage
 
+Usage `WSL (Windows Subsystem for Linux)` can be perfect to run this scripts, otherwise you can only modify the script after using `Git Bash` to run.
+So you'd better install and use `WSL` to run scripts.
```

Comment:
I think this section is not that clear, maybe we can rephrase?

> WSL (Windows Subsystem for Linux)` is a perfect option to run this script otherwise, you can modify the scripts as outlined below to run with other `bash` tools like `Git Bash` or `MSYS2`

Reply:
Yeah, I know both, the main difference for this is just the mount path, that is all. It might be better to script the detection of the environment than to have 2 scripts.